Woche2/caesar/caesar.c

```diff
@@ -0,0 +1,39 @@
+#include <cs50.h>
+#include <stdio.h>
+#include <ctype.h>
+#include <stdlib.h>
+#include <string.h>
+
+int main(int argc, string argv[]) {
+
+    if (argc != 2) {
+        printf("Usage: ./caesar key\n");
+        return 1;
+    }
+
+    for (int i = 0; i < strlen(argv[1]); i++) {
+        
+        if (!isdigit(argv[1][i])) {
```

Comment:
When accessing two dimensional arrays or other "hard to read"-variables it is better to add another line of code to read from it and give that variable a good name like this:

```suggestion
        int someGoodName = argv[1][i];
        if (!isdigit(someGoodName)) {
```